common/src/main/java/com/microsoft/identity/common/internal/logging/DiagnosticContext.java

```diff
@@ -85,4 +94,30 @@ private static boolean hasThreadId() {
     public static void clear() {
         REQUEST_CONTEXT_THREAD_LOCAL.remove();
     }
+
+    /**
+     * Function previously resided in Device.java, contains Constants
+     * Attaching the constants to Diagnostic Context, for the purpose of having them in one place.
```

Comment:
What's being attached to DiagnosticContext here? As far as I can see nothing.....and I think this method could easily stay inside `Device.java`. Is there a specific reason for moving it here?

Reply:
There could be some more code optimizations done, and having all the constants in a single place would help. It would be then easier to have these constants accessible from a single place. Hence setting it up for the later use. @kreedula anything I missed?

Reply:
Well I would like to keep all of it in Device.java and use it in DiagonisticContext to put all the parameters inside this. And later , you get the map from `DiagnosticContext`.